src/reducer/dataReducer.js

```diff
@@ -1,11 +1,27 @@
 export const initialState = {
     product: [],
-    sortBy: ""
+    sortBy: "",
+    rating: "",
+    priceLimit: "",
+    category: [],
 }
 
 export const dataReducer = (state, action) => {
     switch (action.type) {
         case "INIT_PRODUCTS":
             return { ...state, product: action.payload }
+        case "SORT_BY_PRICE":
+            return { ...state, sortBy: action.payload }
+        case "SORT_BY_RATING":
+            return { ...state, rating: action.payload }
+        case "FILTER_BY_PRICE":
+            console.log("pl: ", action.payload);
```

Comment:
Suggestion: better to remove console

Reply:
Done [2053a6c](https://github.com/shivanigangadharan/ecom-app/pull/4/commits/2053a6cca7b4bb2ccdcce9baf4d8ae06ff540bfa)